### DISCUSSION TITLE
Help in Understanding Navigation

### DISCUSSION THREAD

**Discussion by TexanScot** [requarks/wiki#5762] [Help / Questions]
So I have wiki.js running successfully in a docker container on my home server which runs under Ubuntu 18.04 LTS. Installation was a breeze and I really love the interface and the ease of creating pages etc.

Now here's the thing, I generally consider myself a relatively tech savvy individual however I can't for the life of me seem to get my head around organizing, linking and navigating pages. I have a very simple use case; I want to develop a wiki to document information regarding the various hardware, software and applications that I tinker with (home server, raspberry pi's, macbook, docker etc.)  I kind of have in my head that I would have a 'landing' page for each of these categories which would then have links to other topics, e.g. installed software, application configurations, to-do lists, etc.

I know I must be missing something obvious since this strikes me as a ridiculously simple configuration for a tool as powerful as wiki.js. However I have spent more than a few hours scouring the net and can't seem to find what I need to make the concepts click. Can anyone provide a little insight or guidance regarding how I should configure wiki.js to achieve what I want.

**Comment by ICantCodeEtAl** [comment]
What I have done, since this doesn't work like a normal "Tree Wiki" is you a primary page i.e., WEBADRESS.COM/en/Docker. Fill it in with whatever you would place on your landing page. Then you use that as your new folder so your next page would be WEBADRESS.COM/en/Docker/Tips and WEBADRESS.COM/en/Docker/Admin_Info, etc. This will turn your WEBADRESS.COM/en/Docker into a folder and every time you click the folder you will see a "Landing Page" with a list to the side of all the pages stored with in here is where you would place all your links etc. It takes a while to get used to it, it's a change from old practices, we actually have started liking this more.

If I understood your question correctly. Good luck!

**Comment by TexanScot** [comment]
OK, I figured it out! Watched a portion of a Youtube video that showed how to use links to navigate to different pages. Very helpful and exactly what I needed to get started.
